setup.py

```diff
@@ -21,6 +21,7 @@
         'License :: OSI Approved :: Apache Software License',
         'Operating System :: OS Independent',
         'Programming Language :: Python :: 2.7',
+        'Programming Language :: Python :: 3',
```

Comment:
I'm only testing with Python 3.5, Travis runs 3.4. Not sure how these classifiers are actually used but best be specific.